Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:11
```

Comment:
Use openjdk:11-jre-slim for a smaller image
```suggestion
FROM openjdk:11-jre-slim
```

Reply:
done.